.github/workflows/ci.yml

```
@@ -335,12 +335,29 @@ jobs:
           XWIN_ARCH: "x86_64"
           XWIN_CACHE_DIR: "${{ github.workspace }}/.xwin"
 
-      - name: "Build"
+  # Separate jobs for the nightly crate
+  windows-trampoline-test:
+    needs: determine_changes
+    if: ${{ github.repository == 'astral-sh/uv' && (needs.determine_changes.outputs.code == 'true' || github.ref == 'refs/heads/main') }}
+    runs-on: windows-latest
+    name: "test windows trampoline | ${{ matrix.target-arch }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        target-arch: ["x86_64", "i686", "aarch64"]
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Install Rust toolchain"
         working-directory: ${{ github.workspace }}/crates/uv-trampoline
-        run: cargo xwin build --release --target ${{ matrix.target-arch }}-pc-windows-msvc
-        env:
-          XWIN_ARCH: "${{ matrix.target-arch == 'i686' && 'x86' || matrix.target-arch }}"
-          XWIN_CACHE_DIR: "${{ github.workspace }}/.xwin"
+        run: |
+          rustup target add ${{ matrix.target-arch }}-pc-windows-msvc
+          rustup component add rust-src --target ${{ matrix.target-arch }}-pc-windows-msvc
+      - uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: ${{ github.workspace }}/crates/uv-trampoline
+      - name: "Test"
```

```suggestion
      # Build and copy the new binaries
      - name: "Build"
        working-directory: ${{ github.workspace }}/crates/uv-trampoline
        run: |
          cargo build --target ${{ matrix.target-arch }}-pc-windows-msvc
          cp target/${{ matrix.target-arch }}-pc-windows-msvc/debug/uv-trampoline-console.exe trampolines/uv-trampoline-${{ matrix.target-arch }}-console.exe
          cp target/${{ matrix.target-arch }}-pc-windows-msvc/debug/uv-trampoline-gui.exe trampolines/uv-trampoline-${{ matrix.target-arch }}-gui.exe
      - name: "Test"
```

Worth building and copying the new binaries so that they're tested